akka-actor/src/main/scala/akka/event/EventBus.scala

```diff
@@ -180,6 +180,8 @@ trait SubchannelClassification { this: EventBus ⇒
    * Expensive call! Avoid calling directly from event bus subscribe / unsubscribe.
    */
   private[akka] def hasSubscriptions(subscriber: Subscriber): Boolean =
+    // FIXME binary incompatible, but I think it is safe to filter out this problem,
+    //       since it is only called from new functionality in EventStreamUnsubscriber
```

Comment:
Correct, the unsubscriber has to be started somewhere (we do in the ActorSystem for the EventStream), if the user has his own bus they didn't start an unsubscriber - so no-one will call this method